docs/documentation/concepts/items.md

```diff
@@ -137,3 +137,24 @@ Here is a short table demonstrating conversions for the examples above:
 | Color         | `HSBType`      | &bull; `OnOffType` - `OFF` if the brightness level in the `HSBType` equals 0, `ON` otherwise <br/> &bull; `PercentType` - the value for the brightness level in the `HSBType` |
 | Dimmer        | `PercentType`  | `OnOffType` - `OFF` if the brightness level indicated by the percent type equals 0, `ON` otherwise |
 | Rollershutter | `PercentType`  | `UpDownType` - `UP` if the shutter level indicated by the percent type equals 0, `DOWN` if it equals 100, and `UnDefType.UNDEF` for any other value|
+
+## Item Metadata
+
+Sometimes additional information is required to be attached to items for certain use-cases. 
+This could be e.g. an application which needs some hints in order to render the items in a generic way or an integration with voice controlled assistants or any other services which access the items and need to understand their "meaning".
+
+For this purpose, such meta-information can be attached to items using disjunct namespaces so they won't conflict with each other. 
+Each metadata entry has a main value and optionally additional key/value pairs. 
+There can be metadata for as many namespaces attached to an item as desired. 
+The following example show how `something` is configured for the `example` namespace, together with `bar` for the `foo` property in an `*.items`-file:
+
+    Switch light { channel="...", example="something" [ foo="bar" ] }
```

Comment:
I don't like this example. A bit more descriptive would be nice, e.g.: 

The following example shows how `lighting` is configured for the `alexa` namespace, together with `night` for the `timeOfDay` property in an `*.items`-file:
```` 
 Switch light { channel="...", alexa="lighting" [ timeOfDay="night" ] }
````

Reply:
> A bit more descriptive would be nice

Fully agree. I just wanted to prevent any "I have copied the alexa configuration from the documentation and it doesn't work"-situation. But you're right, the example should be improved.